atlite/convert.py

```diff
@@ -493,6 +493,76 @@ def wind(cutout, turbine, smooth=False, **params):
     )
 
 
+# irradiation
+def convert_irradiation(
+    ds, orientation, irradiation="total", trigon_model="simple", clearsky_model="simple"
+):
+    solar_position = SolarPosition(ds)
+    surface_orientation = SurfaceOrientation(ds, solar_position, orientation)
+    irradiation = TiltedIrradiation(
+        ds,
+        solar_position,
+        surface_orientation,
+        trigon_model=trigon_model,
+        clearsky_model=clearsky_model,
+        irradiation=irradiation,
+    )
+    return irradiation
+
+
+def irradiation(
+    cutout, orientation, irradiation="total", clearsky_model=None, **params
+):
+    """
+    Calculate the total, direct, diffuse, or ground irradiation on a tilted
+    surface.
+
+    Parameters
+    ----------
+    orientation : str, dict or callback
+        Surface orientation can be chosen from either
+        'latitude_optimal', a constant orientation {'slope': 0.0,
```

Comment:
Frankly, using `latitude_optimal` doesn't really make much sense to me either. I suppose currently it results in angling the surface so that it collects most of the irradiation over a year, but I'm not sure what it would be used for. My use cases all define a constant known orientation, and I just used the same orientation functions as the PV conversion functions for convenience. I suppose I could clarify the docstring that it shouldn't probably be used here.